pkg/minishift/addon/manager/addon_manager_test.go

```diff
@@ -128,6 +130,14 @@ func Test_invalid_addons_get_skipped(t *testing.T) {
 	expectedNumberOfAddOns := 1
 	actualNumberOfAddOns := len(addOns)
 	if actualNumberOfAddOns != expectedNumberOfAddOns {
-		t.Fatal(fmt.Sprintf("Unexpected number of addons. Expected %d, got %d", expectedNumberOfAddOns, actualNumberOfAddOns))
+		t.Fatal(fmt.Sprintf("Unexpected number of add-ons. Expected %d, got %d", expectedNumberOfAddOns, actualNumberOfAddOns))
 	}
 }
+
+func Test_add_default_envs_to_context(t *testing.T) {
```

Comment:
So why only a test for the happy path? What's about testing the border conditions and the error cases?